content/languages/python/unittest.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Python unittest
+sidebar_label: unittest
+kind: reference
+---
+
+import ThemedImage from "@theme/ThemedImage";
+
+Codewars offers Python's [unittest](https://docs.python.org/3/library/unittest.html) module for testing solutions. Codewars uses a custom reporter for unittest to format the output for the code runner. Visit the [python-test-framework](https://github.com/codewars/python-unittest) GitHub repository for more information.
```

Comment:
We don't _support_ `unittest` to test kata solutions. It's only supported in Kumite. It happens to work if you add `unittest.main(testRunner=CodewarsTestRunner())` because the test is run with `python tests.py`. The runner still does stuff like prepending `import codewars_test as test` because it's asked to use the Codewars test framework.

I really want to make test framework configurable per kata and stop using the custom test framework though.

Reply:
Ah, maybe I got excited too soon! Can I use this to test, for example, Pandas challenges, or is it recommended to stick with CW test framework?

Reply:
I just remembered we discussed this a few years ago. https://github.com/codewars/python-test-framework/pull/7#issuecomment-647050696

I don't want to recommend using `unittest` now, but if you need it, I guess it's fine to use it with a comment at the top explaining the hack.

Reply:
Yep, I saw your recent push in exo and figured it was an option with 3.10.